NameLayerMain/src/vg/civcraft/mc/namelayer/GroupManager.java

```diff
@@ -309,11 +320,26 @@ public boolean hasAccess(String groupname, UUID player, PermissionType perm) {
 			NameLayerPlugin.getInstance().getLogger().log(Level.INFO, "hasAccess failed (access denied), could not find group " + groupname);
 			return false;
 		}
-		
 		GroupPermission perms = getPermissionforGroup(group);
-		PlayerType rank = group.getPlayerType(player);	
-		
-		return perms.isAccessible(rank, perm);
+		for(PlayerType rank : getRecursivePlayerTypes(group, player)) {
+			if (perms.hasPermission(rank, perm)) {
+				//player has right rank in the group itself or at least one super group
+				return true;
+			}
+		}		
+		return false;
+	}
+	
+	private List<PlayerType> getRecursivePlayerTypes(Group group, UUID player) {
```

Comment:
I thought about that, but I would think that from a user perspective it would be more desireable to not have to add people to all the different groups. I intended to let the super group have dominance over the subgroup, where it's not just a linking thing, but a clear ranking order. Anyone who is a mod a on a supergroup would desired to be automatically also at least a mod on the subgroup